space/space.go

```diff
@@ -301,7 +301,7 @@ func (r *GormRepository) listSpaceFromDB(ctx context.Context, q *string, userID
 	}
 
 	// ensure that the result list is always ordered in the same manner
-	db = db.Order("spaces.updated_at DESC")
```

Comment:
is this right approach to change the default behavior of the output or there needs to be some parameter which governs this?

Reply:
this is governed by a parameter so ignore above comment.